curriculum/challenges/english/01-responsive-web-design/applied-accessibility/use-headings-to-show-hierarchical-relationships-of-content.md

```diff
@@ -17,35 +17,35 @@ If you were writing a paper with an introduction, a body, and a conclusion, it w
 
 Headings with equal (or higher) rank start new implied sections, headings with lower rank start subsections of the previous one.
 
-As an example, a page with an `h2` element followed by several subsections labeled with `h4` tags would confuse a screen reader user. With six choices, it's tempting to use a tag because it looks better in a browser, but you can use CSS to edit the relative sizing.
+As an example, a page with an `h2` element followed by several subsections labeled with `h4` elements would confuse a screen reader user. With six choices, it's tempting to use a tag because it looks better in a browser, but you can use CSS to edit the relative sizing.
 
 One final point, each page should always have one (and only one) `h1` element, which is the main subject of your content. This and the other headings are used in part by search engines to understand the topic of the page.
 
 # --instructions--
 
-Camper Cat wants a page on his site dedicated to becoming a ninja. Help him fix the headings so his markup gives semantic meaning to the content, and shows the proper parent-child relationships of his sections. Change all the `h5` tags to the proper heading level to indicate they are subsections of the `h2` ones. Use `h3` tags for the purpose.
+Camper Cat wants a page on his site dedicated to becoming a ninja. Help him fix the headings so his markup gives semantic meaning to the content, and shows the proper parent-child relationships of his sections. Change all the `<h5>` tags to the proper heading level to indicate they are subsections of the `<h2>` ones. Use `<h3>` tags for the purpose.
 
 # --hints--
 
-Your code should have 6 `h3` tags.
+Your code should have 6 `<h3>` tags.
 
 ```js
 assert($('h3').length === 6);
 ```
 
-Your code should have 6 `h3` closing tags.
+Your code should have 6 `<h3>` closing tags.
 
 ```js
 assert((code.match(/\/h3/g) || []).length === 6);
 ```
 
-Your code should not have any `h5` tags.
+Your code should not have any `<h5>` tags.
```

Comment:
As this test is not testing for tags (it is testing for elements), the test text should represent this.

```suggestion
Your code should not have any `h5` elements.
```